docs/source/checks/tabular/model_evaluation/plot_calibration_score.py

```diff
@@ -28,30 +50,17 @@ def custom_formatwarning(msg, *args, **kwargs):
 #%%
 # Binary Classification
 # =====================
-# Load data
-# ---------
-# The dataset is the adult dataset which can be downloaded from the UCI machine
-# learning repository.
-#
-# Dua, D. and Graff, C. (2019). UCI Machine Learning Repository [http://archive.ics.uci.edu/ml].
-# Irvine, CA: University of California, School of Information and Computer Science.
```

Comment:
Why remove?

Reply:
we use the iris and adult datasets in many checks.
To be consistent, we need to remove the above explanation, or add an explanation in all checks.
if you want to add for all, i think that the following explanation is better than the above:

`We will run the check on the adult dataset, which you can download from the [UCI machine learning repository](http://archive.ics.uci.edu/ml), and is also available in deepchecks.tabular.datasets.`

what do you think?

Reply:
@shir22 What do you think? I think it's ok to remove the reference

Reply:
This wording is based on the explanation here:
https://archive.ics.uci.edu/ml/citation_policy.html

(If you publish material based on databases obtained from this repository, then, in your acknowledgements, please note the assistance you received by using this repository + their recommended citation format)

Thus we would want to incorporate it where using these datasets.